dev/maven-init.gradle

```diff
@@ -0,0 +1,12 @@
+// This block configures several source JAR repos for the gradle build
+// to search for dependencies in, in descending order of priority.
+// By default, the components have only the third configured, which is not
```

Comment:
Clarify this. Something like the following

By default, the spinnaker microservices only use jcenter.
For standard spinnaker builds, we want to pick up from the standard spinnaker releases repository first, then from netflix's repository. The netflix repository is included here because the jcenter repository may sometimes lag behind netflix for newly built components (netflix pushes from their repository to jcenter).